src/ui/Sort/Sort.ts

```diff
@@ -51,20 +49,28 @@ export class Sort extends Component {
      * - `@field ascending`/`@field descending`, where you must replace `field` with the name of a sortable field in your index (e.g., `data-sort-criteria="@size ascending"`).
      *
      * You can specify a comma separated list of sort criteria to toggle between when interacting with this component instance (e.g., `data-sort-criteria="date descending,date ascending"`).
-     * Interacting with this compnent instance will cycle through those criteria in the order they are listed in.
+     * You can specify multiple sort criteria to be used in the same request by separating them with a semicolon (e.g., `data-sort-criteria="@size ascending;date ascending"` ).
+     * Interacting with this component instance will cycle through those criteria in the order they are listed in.
      * Typically, you should only specify a list of sort criteria when you want the end user to be able to to toggle the direction of a `date` or `@field` sort criteria.
      * Otherwise, you should configure a distinct `Sort` component instance for each sort criterion you want to make available in your search page.
      *
      * You must specify a valid value for this option in order for this component instance to work correctly.
+     *
+     * Examples:
+     *
+     * - `data-sort-criteria="date ascending"` will create a Sort component that will only allow to sort on `date ascending`, with no possibility to toggle.
+     * - `data-sort-criteria="date ascending, date descending"` will create a Sort component that will allow the end user to toggle between `date ascending` and `date descending` on click.
```

Comment:
creates a Sort component that allows end users to toggle between `date ascending` and `date descending` on click.